src/externs.ts

```diff
@@ -580,6 +599,10 @@ export function generateExterns(
     return [];
   }
 
+  function locationStr(node: ts.Node, namespace: ReadonlyArray<string>) {
```

Comment:
Can you one-liner doc this?  It looks wrong to me but I see it's only for debugging below.

Reply:
Added docs. In my defense, I'm just moving this code around, and you originally added it ;-)